app/jobs/delete_user.rb

```diff
@@ -0,0 +1,14 @@
+class DeleteUser
+  attr_reader :user
+
+  def initialize(user)
+    @user = user
+  end
+
+  def perform
+    profile = User.find(user['id'])
+    email = profile.email
+    profile.destroy
```

Comment:
When you say it like that it does seem it can use a conditional :sweat_smile: 
What would you suggest? Should we just not deliver the confirmation mail when it fails or should we send one informing that it failed?